common-app-lib/src/cmr/common_app/api/launchpad_token_validation.clj

```diff
@@ -1,17 +1,67 @@
 (ns cmr.common-app.api.launchpad-token-validation
   "Validate Launchpad token."
   (:require
+   [cheshire.core :as json]
+   [clojure.data.codec.base64 :as base64]
+   [clojure.string :as string]
    [cmr.common-app.config :as config]
    [cmr.common.services.errors :as errors]
    [cmr.transmit.config :as transmit-config]))
 
+;; Note: Similar code exists at gov.nasa.echo.kernel.service.authentication
 (def URS_TOKEN_MAX_LENGTH 100)
 
+(defn- remove-bearer
+  "Remove the Bearer marker from the token"
+  ([token] (remove-bearer token "Bearer "))
+  ([token prefix]
+   (if (string/starts-with? token prefix)
+     (subs token (count prefix))
+     token)))
+
+(defn- is-legacy-token?
+  "There are two uses cases captured by this test, the Legacy token, the
+   Heritage token (meaning a token made to behave like a legacy token but
+   prefixed with EDL-).
+   Note: Similar code exists at gov.nasa.echo.kernel.service.authentication."
+  [token]
+  (or (string/starts-with? (remove-bearer token) "EDL-")
```

Comment:
why do we remove `Bearer` before checking for `EDL-`? This does not match what is stated in the comment of this function.